connect/middleware.go

```
@@ -0,0 +1,28 @@
+package connect
+
+import (
+	"github.com/micromdm/micromdm/device"
+	"golang.org/x/net/context"
+)
+
+type lastCheckinMiddleware struct {
+	devices device.Datastore
+	next    Service
+}
+
+func NewMiddleware(datastore device.Datastore, next Service) (*lastCheckinMiddleware) {
+	return &lastCheckinMiddleware{
+		datastore,
+		next,
+	}
+}
+
+func (mw lastCheckinMiddleware) Acknowledge(ctx context.Context, req mdmConnectRequest) (int, error) {
+	mw.devices.UpdateDeviceCheckinByUDID(req.UDID)
```

This should not ignore the error return.